packaging/docker/entrypoint.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python3
+import json
+import logging
+import os
+import subprocess
+import time
+from json import JSONDecodeError
+from pathlib import Path
+
+logger = logging.getLogger('monitor')
+logging.basicConfig(level=logging.DEBUG)
+
+DEFAULT_LOG_LEVEL = 'critical'
+
+
+def load_config_from_file():
+    config_file = Path('/config/rotki_config.json')
+    if not config_file.exists():
+        logger.info('no config file provided')
+        return None
+
+    with open(config_file) as file:
+        try:
+            data = json.load(file)
+            return data
+        except JSONDecodeError as e:
+            logger.error(e)
+            return None
+
+
+def load_config_from_env():
+    loglevel = os.environ.get('LOGLEVEL')
+    logfromothermodules = os.environ.get('LOGFROMOTHERMODDULES')
+    sleep_secs = os.environ.get('SLEEP_SECS')
+    max_size_in_mb_all_logs = os.environ.get('MAX_SIZE_IN_MB_ALL_LOGS')
+    max_logfiles_num = os.environ.get('MAX_LOGFILES_NUM')
+
+    return {
+        'loglevel': loglevel,
+        'logfromothermodules': logfromothermodules,
+        'sleep_secs': sleep_secs,
+        'max_logfiles_num': max_logfiles_num,
+        'max_size_in_mb_all_logs': max_size_in_mb_all_logs
+    }
+
+
+def load_config():
+    env_config = load_config_from_env()
+    file_config = load_config_from_file()
+
+    if file_config is not None:
```

Comment:
This makes that the two methods can't be used at the same time. Is that intentional?

Reply:
Yes, it is intentional, do you think it would make sense to allow mix and match? with the file overwriting any other settings?